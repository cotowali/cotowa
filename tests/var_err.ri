let i = 0
let i = 0
