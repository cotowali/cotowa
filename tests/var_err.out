var_err.ri 2,6: `i` is duplicated
